apps/web/app/(app)/environments/[environmentId]/integrations/notion/components/ManageIntegration.tsx

```
@@ -42,7 +42,12 @@ export const ManageIntegration = ({
   const handleDeleteIntegration = async () => {
     try {
       setisDeleting(true);
-      await deleteIntegrationAction({ integrationId: notionIntegration.id });
+      const deleteIntegrationActionResult = await deleteIntegrationAction({
+        integrationId: notionIntegration.id,
+      });
+      if (deleteIntegrationActionResult && deleteIntegrationActionResult.serverError) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)